skll/config/__init__.py

```diff
@@ -263,15 +274,65 @@ def validate(self):
             )
 
 
-def parse_config_file(config_path, log_level=logging.INFO):  # noqa: C901
+def parse_config_file(
+    config_path: PathOrStr, log_level=logging.INFO
+) -> Tuple[
+    str,
+    str,
+    str,
+    List[Dict[str, Any]],
+    bool,
+    int,
+    str,
+    str,
+    str,
+    str,
+    str,
+    List[List[str]],
+    bool,
+    str,
+    bool,
+    List[str],
+    bool,
+    bool,
+    str,
+    str,
+    str,
+    int,
+    str,
+    Optional[int],
+    Optional[Union[int, FoldMapping]],
+    Optional[Union[int, FoldMapping]],
+    int,
+    bool,
+    bool,
+    bool,
+    bool,
+    List[Dict[str, Any]],
+    List[Dict[str, Any]],
+    List[str],
+    List[str],
+    str,
+    str,
+    str,
+    str,
+    bool,
+    Optional[Dict[str, List[str]]],
+    str,
+    str,
+    List[int],
+    List[Number],
+    List[str],
+    bool,
+]:
```

Comment:
Wow, that's a long tuple! wouldn't it be better to use Any in this case?
But more importantly, this doesn't match the return value in the docstring below.

Reply:
Lol, yes there must be but I can't really change this right now since it would be a major refactoring.